Dockerfile

```diff
@@ -0,0 +1,52 @@
+# BUILD:
+# docker build --force-rm=true -t openvpn_exporter .
+
+# RUN:
+# docker run -it -p 9176:9176 -v /path/to/openvpn_server.status:/etc/openvpn_exporter/server.status openvpn_exporter
+
+FROM golang as builder
+
+# Grenoble testing env
+ENV http_proxy="http://web-proxy.gre.hpecorp.net:8080" \
+    https_proxy="http://web-proxy.gre.hpecorp.net:8080"
+
+RUN mkdir /app
+RUN mkdir /go/src/app
+ADD . /go/src/app
+WORKDIR /go/src/app
+
+# Go dep
+RUN go get -u github.com/prometheus/client_golang/prometheus && \
+    go get -u github.com/prometheus/client_golang/prometheus/promhttp
+
+
+# Build a standalone binary
+RUN set -ex && \
+  CGO_ENABLED=0 go build \
+        -tags netgo \
+        -o /app/openvpn_exporter \
+        -v -a \
+        -ldflags '-extldflags "-static"' && \
+  ls
+
+# Create the second stage with a basic image.
+# this will drop any previous
+# stages (defined as `FROM <some_image> as <some_name>`)
+# allowing us to start with a fat build image and end up with
+# a very small runtime image.
+
+FROM busybox
+
+# Grenoble testing env
+ENV http_proxy="http://web-proxy.gre.hpecorp.net:8080" \
+    https_proxy="http://web-proxy.gre.hpecorp.net:8080"
+
+# add compiled binary
+COPY --from=builder /app/openvpn_exporter /openvpn_exporter
+
+# add a default file to be processed
+ADD examples/server2.status /etc/openvpn_exporter/server.status
```

Comment:
These examples are just included in the repository to test what the exporter does. There is no need for including these in any container image.

Reply:
This is just to allow the image to run without any error if the user doesn't provide a file with a volume. As the `-openvpn.status_paths` is mandatory. I can remote it anyway if you prefer.